extensions/src/main/java/org/pentaho/platform/osgi/KarafBoot.java

```diff
@@ -14,7 +14,7 @@
  * See the GNU Lesser General Public License for more details.
  *
  *
- * Copyright (c) 2002-2018 Hitachi Vantara. All rights reserved.
+ * Copyright (c) 2002 - 2020 Hitachi Vantara. All rights reserved.
```

Comment:
We're using spaces?

Reply:
We have places with and others without. Until now both seem to work, so I normally use this version because it is easier to read